helm/nifikop/README.md

```diff
@@ -29,6 +29,8 @@ The following tables lists the configurable parameters of the NiFi Operator Helm
 | `image.imagePullSecrets.name`    | Name of the secret to connect to docker registry                                                                                                                                     | -                        |
 | `certManager.enabled`            | Enable cert-manager integration                                                                                                                                                      | `true`                   |
 | `rbacEnable`                     | If true, create & use RBAC resources                                                                                                                                                 | `true`                   |
+| `commonLabels`                   | Labels to add to all deployed resources                                                                                                                                              | `{}`                     |
```

Comment:
Thoughts on just naming these `labels` and `annotations` and the description clarifies they will be added to every resource in this deployment?

Reply:
I took inspiration from `bitnami/charts` naming conventions 😄 Happy to rename if it's overly verbose

Reply:
Renamed to `labels` and `annotations` on [58485dc](https://github.com/konpyutaika/nifikop/pull/153/commits/58485dc05ade3cf232e367d24d56862b9a6e5285)